### DISCUSSION TITLE
Currency name/type and amount on rewarded ad

### DISCUSSION THREAD

**Discussion by zziime** [Shin-NiL/Godot-Android-Admob-Plugin#119] [Q&A] [answered]
I need some help understanding where to change the currency name and amount. I got the rewarded running well enough on my game, but I'm still not sure where to change these properties. A huge thanks!!

**Comment by zziime** [comment]
Is it actually set on admob dashboard?

**Comment by Shin-NiL** [comment, marked as answer]
> Is it actually set on admob dashboard?

That's it :)
